include/networkit/GlobalState.hpp

```diff
@@ -0,0 +1,94 @@
+#ifndef NETWORKIT_GLOBAL_STATE_HPP_
+#define NETWORKIT_GLOBAL_STATE_HPP_
+
+#ifdef NETWORKIT_STATIC_DEFINE
+#define NETWORKIT_EXPORT
+#define NETWORKIT_NO_EXPORT
+#else
+#ifndef NETWORKIT_EXPORT
+#ifdef networkit_EXPORTS
+#if defined(WIN32)
+#define NETWORKIT_EXPORT __declspec(dllexport)
+#else
+#define NETWORKIT_EXPORT
+#endif
+#else
+#if defined(WIN32)
+#define NETWORKIT_EXPORT __declspec(dllimport)
+#else
+#define NETWORKIT_EXPORT
+#endif
+#endif // networkit_EXPORTS
+#endif // NETWORKIT_EXPORT
+#endif // NETWORKIT_STATIC_DEFINE
```

Comment:
This is quite hard to read without indentation. Is the indentation removed by CMake? Is it also removed if we place it after the `#`?

Reply:
Yes-  and moving the intendation behind `#` does not work.